### PR TITLE
Port support for spaceman 2% milk macropad

### PR DESCRIPTION
**Pull request by Lenbok**
Essentially just ported the existing via keymap for vial, with couple of extra defines to bring the firmware size down to fit on a pro micro.

### CODE REVIEW

File: keyboards/spaceman/2_milk/keymaps/vial/keymap.c [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+          _______,
+          _______
+      ),
+      [2] = LAYOUT(
+          _______,
+          _______
+      ),
+      [3] = LAYOUT(
+          _______,
+          _______
```
Comment (lesshonor) [inline keyboards/spaceman/2_milk/keymaps/vial/keymap.c]: No need to hardcode "filler `KC_TRNS` layers. Dynamic keymaps will automatically fill in "missing" layers at compile-time up to the maximum set in `dynamic_keymap.layer_count` (which you've set to 4).
```suggestion
```

File: keyboards/spaceman/2_milk/keymaps/vial/readme.md
Comment (lesshonor) [inline keyboards/spaceman/2_milk/keymaps/vial/readme.md:1]: This file isn't really necessary.

**Review by lesshonor — APPROVED** [review]

**Comment by Lenbok** [comment]
Thanks for reviewing. I have addressed both of your comments and squashed the commits. Happy for you to merge when ready.

**Review by lesshonor — APPROVED** [review]
Please note that I have no control over merges. All I can do is leave comments on PRs, same as you can.

**Comment by Kuznetsov-s-v** [comment]
I think everything is ready now, and it can be added to the repository. If I have made any mistakes or need to make corrections, please point them out, and I will do so.

**Pull request merged**